FirebaseUI.podspec

```diff
@@ -1,11 +1,12 @@
 Pod::Spec.new do |s|
   s.name         = 'FirebaseUI'
-  s.version      = '0.5.6-rc3'
+  s.version      = '0.5.6-rc4'
   s.summary      = 'UI binding libraries for Firebase.'
   s.homepage     = 'https://github.com/firebase/FirebaseUI-iOS'
   s.license      = { :type => 'Apache 2.0', :file => 'FirebaseUIFrameworks/LICENSE' }
   s.author       = 'Firebase'
-  s.source       = { :http => 'https://github.com/firebase/FirebaseUI-iOS/releases/download/v0.5.6-rc3/FirebaseUIFrameworks.zip' }
+  # s.source       = { :http => 'https://github.com/firebase/FirebaseUI-iOS/releases/download/v0.5.6-rc3/FirebaseUIFrameworks.zip' }
+  s.source = { :http => 'http://localhost:7777/FirebaseUIFrameworks.zip' }
```

Comment:
Yeah, these podspec edits aren't going to be part of the final PR, I just left them around on accident.